packages/notary/actors/notary_actor.py

```diff
@@ -24,8 +31,17 @@ def register(self, robot_id, key, content, content_signature, schema_out):
         ```
         """
         out = schema_out.new()
-        out.message = "your register info is robot_id = {} , key = {}, content = {} ,content_signature ={}".format(
-            robot_id, key, content, content_signature)
+        bcdb = self.bcdb_get()
```

Comment:
Here we need to verify the signature before saving the content.
Then the return value should also be the blak2b hash of the content.